src/ProjectTemplates/test/Templates.Blazor.WebAssembly.Tests/Templates.Blazor.WebAssembly.Tests.csproj

```diff
@@ -6,7 +6,6 @@
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <TestGroupName>ProjectTemplates.Blazor.WebAssembly.Tests</TestGroupName>
     <DefineConstants>$(DefineConstants);XPLAT</DefineConstants>
-    <HelixTimeout>01:00:00</HelixTimeout>
```

Comment:
This increased timeout was [recently introduced](https://github.com/dotnet/aspnetcore/pull/42743) due to these tests timing out. With this split, this should no longer be required.